packages/atomic/src/locales.json

```diff
@@ -476,6 +476,14 @@
     "fr": "Aucun résultat",
     "es-es": "No hay resultados"
   },
+  "moreMatchesFor": {
+    "en": "More matches for {{query}}",
```

Comment:
More a detail, but should we use "matches" or "results"? e.g. `More results ...`, `No results ...` cc: @dlutzCoveo

Reply:
Additional detail: matches is used in the design because we are filtering the values with a exact match, not querying results or searching technically.

Reply:
> More a detail, but should we use "matches" or "results"? e.g. `More results ...`, `No results ...` cc: @dlutzCoveo

I would stick with "matches" given that it is more accurate.